src/main/java/org/owasp/webgoat/server/StartupMessage.java

```diff
@@ -15,16 +15,25 @@ public class StartupMessage {
 
   private String port;
   private String address;
+  private String contextPath;
 
   @EventListener
   void onStartup(ApplicationReadyEvent event) {
-    if (StringUtils.hasText(port)
-        && !StringUtils.hasText(System.getProperty("running.in.docker"))) {
-      log.info("Please browse to http://{}:{}/WebGoat to get started...", address, port);
-    }
+
     if (event.getApplicationContext().getApplicationName().contains("WebGoat")) {
       port = event.getApplicationContext().getEnvironment().getProperty("server.port");
       address = event.getApplicationContext().getEnvironment().getProperty("server.address");
+      contextPath =
+          event.getApplicationContext().getEnvironment().getProperty("server.servlet.context-path");
+    } else if (event.getApplicationContext().getApplicationName().contains("WebWolf")) {
```

Comment:
This branch's code block is the same as the code block of the previous branch, maybe we can refactor that.

Reply:
From the looks of it, this only works for the default context-path. I will make some changes so it will also work for different context-path. I already wondered why i didn’t see the start up message in that case

Reply:
this has been simplified in the last commit